### PR TITLE
E0422 description.

### PR DESCRIPTION
**Pull request by alexispurslane**

### CODE REVIEW

File: src/librustc_typeck/diagnostics.rs [hunk 1]
```diff
@@ -3103,7 +3103,29 @@ parameters. You can read more about it in the API documentation:
...
+struct. For instance:
 
+```
+fn main () {
+    let x = Foo { x: 1, y: 2 };
+}
+```
+
+In this case, `Foo` is undefined, so it inherently isn't anything, and
+definatly not a struct.
```
Comment (apasel422) [inline src/librustc_typeck/diagnostics.rs]: s/definatly/definitely/

**Comment by christopherdumas** [comment]
Thanks, @apasel422! (I also squashed the commits.)

**Comment by apasel422** [comment]
@christopherdumas Should this be in `librustc_resolve/diagnostics.rs` instead?

**Comment by christopherdumas** [comment]
I don't know.

**Comment by apasel422** [comment]
`git grep E0422` indicates the the error originates from that file, so presumably yes.

**Comment by christopherdumas** [comment]
Yeah, your right.

**Comment by christopherdumas** [comment]
I squashed the commits again.

**Comment by apasel422** [comment]
You'll also have to remove [line 914](https://github.com/christopherdumas/rust/blob/E0422/src/librustc_resolve/diagnostics.rs#L914).

**Comment by christopherdumas** [comment]
Whoops! OK.

File: src/librustc_resolve/diagnostics.rs [hunk 2]
```diff
@@ -888,7 +911,7 @@ register_diagnostics! {
     E0418, // is not an enum variant, struct or const
     E0420, // is not an associated const
     E0421, // unresolved associated const
-    E0422, // does not name a structure
+//  E0422, // does not name a structure
```
Comment (apasel422) [inline src/librustc_resolve/diagnostics.rs]: You can delete it, rather than comment it out.

Comment (christopherdumas) [inline src/librustc_resolve/diagnostics.rs]: OK? Is that better? I noticed that some are commented out, while some are deleted.

**Comment by apasel422** [comment]
That's better. The ones that are commented out are commented out instead of deleted in order to ensure that no one uses them in the future (because they apply to errors that no longer exist).

**Pull request merged**